tests/ci/profile_handler.go

```diff
@@ -215,25 +213,35 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 	if profile.BYOVPC {
 		var zones []string
 		var vpcOutput *EXE.VPCOutput
-		if profile.Zones != "" {
-			zones = strings.Split(profile.Zones, ",")
-		}
-
-		vpcOutput, err = PrepareVPC(profile.Region, profile.PrivateLink, profile.MultiAZ, zones, clusterArgs.ClusterName)
-		if err != nil {
-			return
-		}
-		clusterArgs.AWSAvailabilityZones = vpcOutput.AZs
-		if vpcOutput.ClusterPrivateSubnets == nil {
-			err = fmt.Errorf("error when creating the vpc, check the previous log. The created resources had been destroyed")
-			return
-		}
-		if profile.PrivateLink {
-			clusterArgs.AWSSubnetIDs = vpcOutput.ClusterPrivateSubnets
+		// Supports ENV set passed to make cluster provision more flexy in prow
+		// Export the subnetIDs via env variable if you have existing ones export SubnetIDs=<subnet1>,<subnet2>,<subnet3>
+		// Export the availability zones via env variable export AvailabilitiZones=<az1>,<az2>,<az3>
+		if os.Getenv("SubnetIDs") != "" && os.Getenv("AvailabilitiZones") != "" {
```

Comment:
@yasun1 No, there is one already to overwrite the profile by ENV those are highlevel. This part is only exposed for shared vpc in case anybody wants to pass the value from other steps.